cli/exec.go

```diff
@@ -145,9 +146,12 @@ func holdHijackConnection(ctx context.Context, conn net.Conn, reader *bufio.Read
 	go func() {
 		if stdin {
 			io.Copy(conn, os.Stdin)
+			// close write if receive CTRL-D
+			if cw, ok := conn.(ioutils.CloseWriter); ok {
```

Comment:
Do you mean `Ctr-D` is detach key?

Reply:
CTRL-D will send the EOF to close stdin. In here, we should close the writer not whole the connection, because we need to receive the data from reader.